tms/user_apis.py

```diff
@@ -0,0 +1,31 @@
+"""All the Client APIs."""
+
+from django.contrib.auth import authenticate
+from rest_framework.authentication import TokenAuthentication
+from rest_framework.authtoken.models import Token
+from rest_framework.parsers import JSONParser
+from rest_framework.permissions import AllowAny, IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+
+class LoginView(APIView):
+    """Class based view loggin in user and returning Auth Token."""
+
+    authentication_classes = [TokenAuthentication]
+    permission_classes = [AllowAny]
+
+    def post(self, request):
+        """Check if user exists, return token if it does."""
```

Comment:
You are not using serializers here ?
They will do the all kinds of validation for you.

Reply:
I am using them for models but not here as there are only two fields. 
But sure, I will add & use serializes here too, to be consistent with other overall code.